handlers/holiday-stop-api/src/main/scala/com/gu/holiday_stops/Handler.scala

```diff
@@ -103,9 +105,9 @@ object Handler extends Logging {
     for {
       sfClient <- SalesforceClient.auth(response, config.sfConfig) match {
         case Right(success) => ContinueProcessing(success)
-        case Left(errors) if errors.exists(_.errorCode == "INVALID_OPERATION_WITH_EXPIRED_PASSWORD") =>
+        case Left(errors) if errors.exists(errors => csrAuthErrors.contains(errors.errorCode)) =>
```

Comment:
Tiny nitpick, but in the predicate shouldn't the yielded value be singular?

```suggestion
        case Left(errors) if errors.exists(error => csrAuthErrors.contains(error.errorCode)) =>
```